cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -4229,6 +4229,14 @@ fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
             }
         }
 
+        Opcode::TlsValue => {
+            let dst = get_output_reg(ctx, outputs[0]).only_reg().unwrap();
+            let (name, _, _) = ctx.symbol_value(insn).unwrap();
+            let symbol = name.clone();
+            ctx.emit(Inst::ElfTlsGetAddr { symbol });
```

Comment:
Can you add a `todo!()` for non elf tls?

Reply:
Done!